src/main/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagApp.scala

```diff
@@ -57,13 +58,32 @@ class EasyValidateDansBagApp(configuration: Configuration) extends DebugEnhanced
   }
 
   def validate(uri: URI, infoPackageType: InfoPackageType): Try[ResultMessage] = {
-    for {
+    val bagName = File(uri).name
+    logger.info(s"start validating bag: $bagName")
```

Comment:
can you format it like this: 
[<uuid>] <The actual log message>
that way, we can easily find all message pertaining to the same deposit

Reply:
will do